site/src/sphinx/setup-configuration.rst

```diff
@@ -17,9 +17,14 @@ defaults:
             "host": "*",
             "port": 36462
           },
-          "protocol": "http"
+          "protocol": "https"
```

Comment:
How about keeping this code block identical with the default `dogma.json` and adding a dedicated section for configuring TLS instead?

Reply:
OK!